cmd/osv-scanner/main.go

```diff
@@ -34,6 +34,14 @@ func run(args []string, stdout, stderr io.Writer) int {
 		Writer:    stdout,
 		ErrWriter: stderr,
 		Flags: []cli.Flag{
+			&cli.BoolFlag{
+				Name:  "experimental-compare-locally",
```

Comment:
nit: let's just name these `experimental-local-db` and `experimental-offline` instead. The offline flag can be more general as an explicit airgapped mode.  

side note: We may also want to add support for a `OSV_LOCAL_DB_PATH` env var or `--local-db-path` flags at some point to give users more control to support airgapped use cases.